draft-johani-dnsop-transport-signaling-02.md

```diff
@@ -148,40 +148,47 @@ capitals.
 # 3. Modes of Operation
 
 [new text, needs to be integrated]
-
-This document provides two modes, opportunistic mode and strict mode.
+<!--
+Note: Tweaks by johani here.
+-->
+This document provides two modes of DNS transport signaling: opportunistic
+mode and strict mode. Both modes are based on the authoritative nameserver
+providing a transport signal via an SVCB record in a DNS reponse. Which mode
```

Comment:
My reasoning is that de draft places requirements on both authoritative and recursive for opportunistic mode and for strict mode. Operators can choose what mode they will support.